### PR TITLE
LPD-30131 Refactor Marketplace tests to use helpers

### PR DESCRIPTION
**Pull request by joanagrego**

### CODE REVIEW

**Comment by joanagrego** [comment]
ci:test:sf

File: modules/test/playwright/tests/workspaces/liferay-workspace-marketplace/pages/customerDashboardAppDetailsPage.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
```
Comment (welitonpe) [inline modules/test/playwright/tests/workspaces/liferay-workspace-marketplace/pages/customerDashboardAppDetailsPage.ts:9]: Header is duplicated

**Review by welitonpe — APPROVED** [review]
Good Job!!! :fire:

**Comment by joanagrego** [comment]
ci:test:sf

**Comment by joanagrego** [comment]
ci:test:playwright

**Comment by joanagrego** [comment]
ci:stop

**Comment by joanagrego** [comment]
ci:test:sf

**Comment by joanagrego** [comment]
ci:test:sf

**Comment by joanagrego** [comment]
ci:test:playwright

**Comment by joanagrego** [comment]
ci:test:playwright

**Comment by joanagrego** [comment]
ci:stop

**Comment by joanagrego** [comment]
ci:test:playwright:nocompile

**Comment by joanagrego** [comment]
ci:reevaluate:1257401_5681

**Comment by joanagrego** [comment]
ci:forward

**Comment by joanagrego** [comment]
ci:test:relevant

**Comment by joanagrego** [comment]
Resent --> [PR](https://github.com/is-solutions-delivery/liferay-portal/pull/5858)

**Pull request closed without merging**